src/assets/text/scrollyText.js

```diff
@@ -6,71 +6,67 @@ export default {
+          text: 'When soil moisture becomes depleted, agricultural drought sets in. To keep crops and livestock healthy, farmers use more water for irrigation.'
         },
         {
           // id is used to source image files with the same naming
           // and used to assign classes to scrolling text
           id: 'aa04',
-          text: 'text aa04: Prolonged agricultural drought can begin to cause hydrological drought, which affects deeper soil layers and eventually reduces streamflow. Streamflow drought is one type of hydrological drought.'
+          text: 'As these conditions persist, less water moves into and through streams. When streamflow levels are unusually low, this is defined as streamflow drought. But what does "unusually low" really mean?'
         },
         {
           // id is used to source image files with the same naming
           // and used to assign classes to scrolling text
           id: 'a',
-          text: 'text a: Although we have been hearing a lot about drought in the western US lately, drought can happen anywhere. For example, here\'s one severe streamflow drought that occurred at a site in Ohio in late summer of 1963.'
+          text: 'Consider this Ohio streamgage site in the late summer of 1963. On August 15th there was a severe streamflow drought.'
         },
         {
           id: 'b',
-          text: 'text b: Streamflow droughts like this occur whenever streamflow gets below a certain threshold. The threshold can be defined based on typical streamflow rates over the year or by day. Let\’s explore how changing the threshold affects when we detect streamflow droughts at this site.'
+          text: 'We can call streamflow unusually low whenever streamflow drops below a certain level or threshold. Here, drought happened when flow dropped below 45 cubic feet per second (cfs). Why 45 cfs? There are two methods to choose that threshold, depending on the questions we are trying to answer.'
         },
         {
           id: 'c',
-          text: 'text c: To begin, start with a hydrograph, which is a plot of streamflow over time. Here, we are looking at daily streamflow during the growing season of 1963. '
+          text: 'To begin, we need to consider historical conditions. Over the past 70 years at this site in Ohio, average daily streamflow ranged from 100 to 2700 cfs, with August through October typically having the lowest streamflow during the year.'
         },
         {
           id: 'd',
-          text: 'text d: A \"fixed\" threshold is created by first calculating average streamflow across all years and days and then calculating 10% of this annual average streamflow as the threshold.'
+          text: 'By considering records across the whole year, streamflow levels that fall below a threshold of 45 cfs are in the lowest 10%. This fixed threshold of 45 cfs is constant from day-to-day and identifies drought events only during the dry season when the lowest streamflow levels occur.'
```

Comment:
```suggestion
          text: 'When considering records across the whole year, streamflow levels that fall below a threshold of 45 cfs are in the lowest 10%. This fixed threshold of 45 cfs is constant from day-to-day and identifies drought events only during the dry season when the lowest streamflow levels occur.'
```

Reply:
^ SC

Reply:
Strong SC: I'm unsure about the text that says this method 'only' identifies droughts during the dry season, unless we specify that it's for this site + year. In the drought timelines we've made w/ the fixed threshold data there are droughts identified throughout the year -- I think it's just that this method primarily identifies droughts during the dry season